<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BwaSettings</class>
 <widget class="QWidget" name="BwaSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>424</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QRadioButton" name="maxDiffRadioButton">
        <property name="text">
         <string>Max #diff (-n)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="maxDiffSpinBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="missingProbRadioButton">
        <property name="text">
         <string>Missing prob (-n)</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="missingProbSpinBox">
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
        <property name="value">
         <double>0.040000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Max gap opens (-o)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="margin">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="maxGapOpensSpinBox">
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Index algorithm (-a)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="margin">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="indexAlgorithmComboBox">
        <property name="currentIndex">
         <number>2</number>
        </property>
        <item>
         <property name="text">
          <string>bwtsw</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>div</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>is</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="1" column="1">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string/>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0" colspan="2">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Threads (-t)</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="2" colspan="2">
          <widget class="QSpinBox" name="threadsSpinBox">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="value">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Mismatch penalty (-M)</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="2" colspan="2">
          <widget class="QSpinBox" name="mismatchPenaltySpinBox">
           <property name="maximum">
            <number>2000000</number>
           </property>
           <property name="value">
            <number>3</number>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Gap open penalty (-O)</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="2" colspan="2">
          <widget class="QSpinBox" name="gapOpenPenaltySpinBox">
           <property name="maximum">
            <number>2000000</number>
           </property>
           <property name="value">
            <number>11</number>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="2">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>Gap extension penalty (-E)</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="3" column="2" colspan="2">
          <widget class="QSpinBox" name="gapExtensionPenaltySpinBox">
           <property name="maximum">
            <number>2000000</number>
           </property>
           <property name="value">
            <number>4</number>
           </property>
          </widget>
         </item>
         <item row="4" column="0" colspan="2">
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>Best hits (-R)</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="4" column="2" colspan="2">
          <widget class="QSpinBox" name="bestHitsSpinBox">
           <property name="maximum">
            <number>2000000</number>
           </property>
           <property name="value">
            <number>30</number>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>Quality threshold (-q)</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="5" column="2" colspan="2">
          <widget class="QSpinBox" name="qualityThresholdSpinBox">
           <property name="maximum">
            <number>255</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string/>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Max gap extensions (-e)</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="maxGapExtensionsSpinBox">
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>2000000</number>
           </property>
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Indel offset (-i)</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="indelOffsetSpinBox">
           <property name="maximum">
            <number>2000000</number>
           </property>
           <property name="value">
            <number>5</number>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Max long deletion extensions (-d)</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSpinBox" name="maxLongDeletionExtensionsSpinBox">
           <property name="maximum">
            <number>2000000</number>
           </property>
           <property name="value">
            <number>10</number>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Seed length (-l)</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QSpinBox" name="seedLengthSpinBox">
           <property name="maximum">
            <number>2000000</number>
           </property>
           <property name="value">
            <number>32</number>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_8">
           <property name="layoutDirection">
            <enum>Qt::RightToLeft</enum>
           </property>
           <property name="text">
            <string>Max seed differences (-k)</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QSpinBox" name="maxSeedDifferencesSpinBox">
           <property name="maximum">
            <number>2000000</number>
           </property>
           <property name="value">
            <number>2</number>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Max queue entries (-m)</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QSpinBox" name="maxQueueEntriesSpinBox">
           <property name="maximum">
            <number>2000000</number>
           </property>
           <property name="value">
            <number>2000000</number>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>Barcode length (-B)</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QSpinBox" name="barcodeLengthSpinBox">
           <property name="minimum">
            <number>-2000000</number>
           </property>
           <property name="maximum">
            <number>2000000</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="enableLongGapsCheckBox">
        <property name="text">
         <string>Enable long gaps</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>groupBox_2</zorder>
     <zorder>groupBox_3</zorder>
     <zorder>enableLongGapsCheckBox</zorder>
    </widget>
   </item>
   <item row="2" column="0" rowspan="2">
    <widget class="QGroupBox" name="groupBox_5">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_7">
      <item row="0" column="0">
       <widget class="QCheckBox" name="colorspaceCheckBox">
        <property name="text">
         <string>Colorspace (-c)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="longScaledGapPenaltyForLongDeletionsCheckBox">
        <property name="text">
         <string>Long-scaled gap penalty for long deletions (-L)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="nonIterativeModeCheckBox">
        <property name="text">
         <string>Non-iterative mode (-N)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>maxDiffRadioButton</tabstop>
  <tabstop>maxDiffSpinBox</tabstop>
  <tabstop>missingProbRadioButton</tabstop>
  <tabstop>missingProbSpinBox</tabstop>
  <tabstop>maxGapOpensSpinBox</tabstop>
  <tabstop>indexAlgorithmComboBox</tabstop>
  <tabstop>maxGapExtensionsSpinBox</tabstop>
  <tabstop>indelOffsetSpinBox</tabstop>
  <tabstop>maxLongDeletionExtensionsSpinBox</tabstop>
  <tabstop>seedLengthSpinBox</tabstop>
  <tabstop>maxSeedDifferencesSpinBox</tabstop>
  <tabstop>maxQueueEntriesSpinBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>enableLongGapsCheckBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>maxGapExtensionsSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>237</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>maxDiffRadioButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>maxDiffSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>14</y>
    </hint>
    <hint type="destinationlabel">
     <x>237</x>
     <y>14</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>maxDiffRadioButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>missingProbSpinBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>15</y>
    </hint>
    <hint type="destinationlabel">
     <x>237</x>
     <y>42</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>missingProbRadioButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>maxDiffSpinBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>237</x>
     <y>15</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>missingProbRadioButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>missingProbSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>237</x>
     <y>40</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
